### ISSUE TITLE
CEA Component in Grasshopper: the content of the .cpg file needs to be "ISO-8859-1"

### ISSUE THREAD

**Issue by shizhongming** [architecture-building-systems/CityEnergyAnalyst#1034]
Of all the affiliated files to the shapefile. We have to make sure that in the file named zone.cpg, the content should be "ISO-8859-1". If the content inside is different, need to replace it with "ISO-8859-1".

![8](https://user-images.githubusercontent.com/16640327/35791665-3455169c-0a84-11e8-85bd-29cea4eb4dc4.PNG)

**Comment by daren-thomas** [comment]
Probably a better idea to figure out how to make sure it's generated as ISO-8859-1 - btw: why does it have to be this codepage again? There was some error associated with this, right? @shizhongming Could you please add that error to the issue?

**Comment by daren-thomas** [comment]
@shizhongming I need more convincing that this is an error. Anyway. On my machine, it produces an ISO-8859-1. I can specify any encoding as needed. e.g. "utf-8" by adding an `encoding='utf-8'` parameter to the `to_file` method in `excel_to_shapefile`:

```
gdf.to_file(shapefile, driver='ESRI Shapefile', encoding='utf-8')
```

NOTE: changing the value in the `.cpg` file manually is a seriously bad idea. In the case of `ISO-8859-1` and `ISO-8859` this should not be a problem most of the times because the code pages are similar. But that doesn't have to be the case.

I have committed a proposed fix for this issue in commit ead34eb3eaf669f13e186e32e39159dae045a4c4 - again, just synchronizing the branch should get you the fix. This makes sure all shapefiles are written with that code page.

**Comment by shizhongming** [comment]
Thanks @daren-thomas . For this issue, the CEA component works perfect now. We can close this issue. Thx!
